apps/comps/e2e/tests/competition-queries.test.ts

```diff
@@ -155,204 +113,19 @@ describe("Competition API", () => {
     });
 
     // Should still be able to get rules after competition is started
-    const activeRulesResponse = (await agentClient.getCompetitionRules(
+    const activeRules = await rpcClient.competitions.getRules({
       competitionId,
-    )) as CompetitionRulesResponse;
-
-    expect(activeRulesResponse.success).toBe(true);
-    expect(activeRulesResponse.rules).toBeDefined();
-    expect(activeRulesResponse.competition.status).toBe("active");
-  });
-
-  test("agents can view competition status and leaderboard", async () => {
```

Comment:
There are some tests that are fully deleted in this PR.  This is generally because they are testing the same endpoints that are tested elsewhere, but testing that they are accessible with different authentication methods.  Given all the endpoints used in this file are public and have no access control, there's not any concern about the different auth methods behaving differently, so I removed the redundant tests to simplify things.